File: src/imagebuilder/flyingtux/naming.py
# SPDX-License-Identifier: AGPL-3.0-or-later
"""
helper functions for naming conventions. the whole purpose of
them is moving them all to a central place for better clarity.
"""

def app_container_name(appname, version):
    return 'flyingtux-app-%s_%s' % (appname, version)

File: internal/util/naming.go
// SPDX-License-Identifier: AGPL-3.0-or-later

package util

import (
	"fmt"
	"path/filepath"
	"strings"
)

// AppContainerName builds the runtime container name for an app image,
// ported from flyingtux.naming.app_container_name.
func AppContainerName(appname, version string) string {
	return fmt.Sprintf("flyingtux-app-%s_%s", appname, version)
}

// CleanPath normalizes a path and strips any "../" components
// (flyingtux.util.clean_path).
func CleanPath(path string) string {
	return strings.ReplaceAll(filepath.Clean(path), "../", "")
}
